mkt/webpay/webpay_jwt.py

```diff
@@ -77,6 +87,20 @@ def external_id(self):
     def name(self):
         return self.webapp.name
 
+    def default_locale(self):
+        return self.webapp.default_locale
+
+    def localized_properties(self):
+        props = {}
+
+        for attr in ('name', 'description'):
+            tr_object = getattr(self.webapp, attr)
+            for tr in Translation.objects.filter(id=tr_object.id):
```

Comment:
mmm, yeah, lots of worms, lots of cans. the FIXMEs in ``translations/models.py`` are scary. I couldn't see of a more direct way to do this but I was hoping @diox might be able to take a look. When manifests are updated we do at least [attempt to delete transactions](https://github.com/mozilla/zamboni/blob/master/mkt/webapps/models.py#L871).